Symfony/CS/Fixer/PSR0/Psr0Fixer.php

```diff
@@ -161,6 +160,12 @@ public function supports(\SplFileInfo $file)
             return false;
         }
 
+        $tokens = token_get_all(sprintf('<?php %s', $filenameParts[0]));
+        if ($tokens[1][0] !== T_STRING) {
+            // do not allow PHP keywords as classy name
```

Comment:
yes,  otherwise the fixer will generate classes like this
```php
<?php 
// some name space

class __CLASS__ {

}
```
which is not valid php